server/interceptor/ratelimiter/limiter.go

```diff
@@ -73,10 +72,10 @@ func (this *Limiter) Handle(w http.ResponseWriter, r *http.Request) error {
 		return nil
 	}
 
-	tollbooth.SetResponseHeaders(this.tbLimiter, w)
-	httpError := tollbooth.LimitByRequest(this.tbLimiter, r)
+	httplimiter.SetResponseHeaders(this.httpLimiter, w)
+	httpError := httplimiter.LimitByRequest(this.httpLimiter, r)
 	if httpError != nil {
-		w.Header().Add("Content-Type", this.tbLimiter.MessageContentType)
+		w.Header().Add("Content-Type", this.httpLimiter.ContentType)
 		w.WriteHeader(httpError.StatusCode)
 		w.Write(util.StringToBytesWithNoCopy(httpError.Message))
 		util.Logger().Warn("Reached maximum request limit!", nil)
```

Comment:
please log print the limit remote ip and request URI

Reply:
fixed